pairtools/lib/scaling.py

```diff
@@ -329,70 +330,60 @@ def compute_scaling(
     regions=None,
     chromsizes=None,
     dist_range=(int(1e1), int(1e9)),
-    n_dist_bins=8 * 8,
+    n_dist_bins_decade=8 * 8,
     chunksize=int(1e7),
     ignore_trans=False,
     keep_unassigned=False,
     filter_f=None,
     nproc_in=1,
-    cmd_in=None,
 ):
     """
-    Main function for computing scaling.
+    Compute the contact-frequency-vs-distance (aka "scaling") curve from a table of contacts.
 
     Parameters
     ----------
-    pairs: pd.DataFrame, stream of fiel paht with pairs.
-    regions: bioframe viewframe, anything that can serve as input to bioframe.from_any, or None
-    chromsizes: additional dataframe with chromosome sizes, if different from regions
-    dist_range: (int, int) tuple with distance ranges that will be split into windows
-    n_dist_bins: number of logarithmic bins
-    chunksize: size of chunks for calculations
-    ignore_trans: bool, ignore trans or not
-    keep_unassigned: bool, keep pairs that are not assigned to any region
-    filter_f: filter function that can be applied to each chunk
-    nproc_in
-    cmd_in
+    pairs : pd.DataFrame or str or file-like object
+        A table with pairs of genomic coordinates representing contacts. 
+        It can be a pandas DataFrame, a path to a pairs file, or a file-like object.
+    regions : bioframe viewframe or None, optional
+        Genomic regions of interest. It can be anything that can serve as input to bioframe.from_any,
+        or None if not applicable.
+    chromsizes : pd.DataFrame or None, optional
+        Additional dataframe with chromosome sizes, if different from regions.
+    dist_range : tuple of int, optional
+        The range of distances to calculate the scaling curve. Default is (10, 1000000000).
+    n_dist_bins : int, optional
```

Comment:
needs to be updated to n_dist_bins_decade?

Reply:
ah, sure! can i update it after merging? :)

Reply:
done!